plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/ArtifactHandler.kt

```diff
@@ -142,51 +144,51 @@ class ArtifactHandler(private val project: Project, private val clientAdaptor: G
 
             CodetransformTelemetry.vcsDiffViewerVisible(
                 codeTransformSessionId = CodeTransformTelemetryState.instance.getSessionId(),
-                codeTransformJobId = jobId.id
+                codeTransformJobId = jobId.id,
```

Comment:
nit: same as above

Reply:
i'm fine with the trailing comma, we should enforce trailing as part of lint